gufe/custom_typing.py

```diff
@@ -8,6 +8,6 @@
 try:
     from typing import TypeAlias  # type: ignore
 except ImportError:
-    from typing_extensions import TypeAlias
+    from typing import TypeAlias
```

Comment:
lol, we can now remove this little code block since 
```
try:
    from typing import TypeAlias  # type: ignore
except ImportError:
    from typing import TypeAlias
```
is just madness

Reply:
"well sure it didn't work, but try it again!"